src/test_encode_decode/mod.rs

```diff
@@ -38,7 +41,7 @@ fn fill_frame<T: Pixel>(ra: &mut ChaChaRng, frame: &mut Frame<T>) {
   }
 }
 
-pub(crate) fn read_frame_batch<T: Pixel>(
+fn read_frame_batch<T: Pixel>(
```

Comment:
Just out of curiosity, why did the visibility of this function change? Not seeing it called anywhere.

Reply:
Well, it is perhaps out of scope of this PR, but I noticed that it didn't need `pub(crate)` so I switched it to `pub`.

Reply:
I think the preference is for `pub(crate)` as we want to expose less API outside of the crate.

Reply:
Wait. I switched it to just `fn`, it's less visibility than `pub(crate)`.

Reply:
> Wait. I switched it to just `fn`, it's less visibility than `pub(crate)`.

Aha, I should have read your code and not your comment.